pom.xml

```diff
@@ -96,25 +96,25 @@
           <target>1.${java.level}</target>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>animal-sniffer-maven-plugin</artifactId>
-        <version>1.14</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>check</goal>
-            </goals>
-          </execution>
-        </executions>
-        <configuration>
-          <signature>
-            <groupId>org.codehaus.mojo.signature</groupId>
-            <artifactId>java1${java.level}</artifactId>
-            <version>1.0</version>
-          </signature>
-        </configuration>
-      </plugin>
+      <!--<plugin>-->
```

Comment:
Animal Sniffer complains that the `java.lang.invoke.MethodHandle#invoke` is missing. Not sure what we can do?